src/dsp_tools/commands/excel2json/new_lists/compliance_checks.py

```diff
@@ -230,12 +233,12 @@ def _check_for_missing_translations_all_excels(sheet_list: list[ExcelSheet]) ->
 
 def _check_for_missing_translations_one_sheet(sheet: ExcelSheet) -> MissingTranslationsSheetProblem | None:
     col_endings = [str(num) for num in get_hierarchy_nums(sheet.df.columns)]
-    col_endings.append("list")
     languages = get_all_languages_for_columns(sheet.df.columns)
     all_cols = _compose_all_combinatoric_column_titles(col_endings, languages)
     problems = []
-    for column_group in all_cols:
-        problems.extend(_check_for_missing_translations_one_column_level(column_group, sheet.df))
+    for column_group in all_cols.reverse_sorted_node_cols():
+        problems.extend(_check_for_missing_translations_one_column_level(column_group.columns, sheet.df))
+    problems.extend(_check_for_missing_translations_one_column_level(all_cols.list_cols.columns, sheet.df))
```

Comment:
I don't understand why this additional line (after the for loop) is necessary.

Reply:
Because i loop over the columns of the nodes, and then I check the columns for the list itself. These are separate because of a change that I am planning on in the future. Basically the big picture plan here is that I want to change how the nodes are deserialised. This should then make the comments easier. But again this will be a big diff and i am trying to cut it up. So it is possible that some things seem a bit weird for the moment.